Library/DasharoSystemFeaturesUiLib/DasharoSystemFeaturesStrings.uni

```diff
@@ -66,3 +69,11 @@
 
 #string STR_WATCHDOG_TIMEOUT_PROMPT   #language en-US  "Watchdog timeout value"
 #string STR_WATCHDOG_TIMEOUT_HELP     #language en-US  "Sets the watchdog timer expiration value in seconds. Minimum is 60s, maximum 1024s."
+
+#string STR_FAN_CURVE_PROMPT		#language en-US "Fan profile"
+#string STR_FAN_CURVE_HELP			#language en-US "Sets the preferred fan operation mode.\n\n"
+													"Silent: The fan will operate at lower speeds to reduce noise.\n\n"
+													"Performance: The fans will work at higher speeds to reduce temperature and increase performance."

```

Comment:
The indentation here is really off, please fix @mkopec

Reply:
Looks like a mismatch between how github and my editor render tabs, it renders okay for me. I was trying to align it like the ME options above

Reply:
@mkopec not exactly... You have used hard tabs, while EDK2 indent convention is to use two spaces. I have already pushed a commit that fixes it.

Reply:
Oh ok, thanks.